src/main/resources/application.yml

```diff
@@ -9,16 +9,18 @@ spring:
   application:
     name: jwt-demo
   datasource:
-      username: jwtuser
-      password: letmein
-      url: jdbc:postgresql://localhost:5432/jwt-db
+    username: jwtuser
+    password: aFFsaToBgb1PXTrn2xAZHmBBK9zAcTc0
```

Comment:
@AbdullatifHabiba Leaving password like this is extremely harm, especially if you put a DB cloud instance like the one you configured. better to use ENV from spring 
`password: ${DB_PASSWORD:local_password}`

Reply:
@sweelam 
Thank you for your valuable feedback and for pointing out the security risk associated with leaving passwords in the configuration files. I apologize for my delayed response; I was busy with my final exams.

I understand the risks of exposing sensitive information, especially when using a cloud-based database instance. For this particular instance, I included the password directly in the application.yml file to facilitate local testing of the pipeline without needing additional configuration. However, I recognize the importance of best practices and will ensure that such information is handled securely in the future.
**I'm eager to learn from any additional notes or advice you may have.**

Reply:
Does it mean you don't have issue to expose the DB credentials publicly? 
Please note in local-dev we need to use docker instead of such DB, this is what we call "Dev/Prod" parity 
Kindly update the code in order to merge your PR.

Reply:
Thanks, Eng @sweelam , for your reply. I wanted to let you know that I've made updates to the workflow to automate Docker image builds and create a test PostgreSQL database image for pipeline testing whenever there are new updates to the repository. However, I have a question regarding whether continuous deployment (CD) is intended solely for local development or if it can be expanded to include other environments as well.